sia/components/renter.go

```diff
@@ -0,0 +1,14 @@
+package components
+
+import (
+// "github.com/NebulousLabs/Sia/consensus"
+)
+
+type RenterUpdate struct {
+	HostDB HostDB
+}
+
+type Renter interface {
+	// UpdateRenter changes the settings used by the host.
+	UpdateRenter(RenterUpdate) error
+}
```

Comment:
This design is a bit confusing to me. I would expect the `Renter` to have a `Rent` method. Also, instead of a single `UpdateRenter` method, I would expect a separate method for each setting, like `UpdateHostDB`.